spring-cloud-dataflow-core/src/main/java/org/springframework/cloud/dataflow/core/TaskManifestConverter.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.dataflow.core;
+
+import javax.persistence.AttributeConverter;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+
+import org.springframework.cloud.deployer.resource.maven.MavenProperties;
+import org.springframework.cloud.deployer.spi.core.AppDefinition;
+import org.springframework.cloud.deployer.spi.core.AppDeploymentRequest;
+import org.springframework.core.io.DefaultResourceLoader;
+import org.springframework.core.io.Resource;
+
+/**
+ * @author Michael Minella
+ */
+public class TaskManifestConverter implements AttributeConverter<TaskExecutionManifest.Manifest, String> {
+
+	private ObjectMapper objectMapper;
+
+	public TaskManifestConverter() {
+		this.objectMapper = new ObjectMapper();
```

Comment:
We've been trying to use a shared single object mapper which gets configured in a boot way and given to us from a boot. This way things around it are not scattered in different places.

Reply:
Can you make a JPA converter a bean so that you can use DI?  If so, I'd be interested in learning how.

Reply:
Shouldn't it just work as bean per github.com/spring-projects/spring-framework/issues/20852

Reply:
Thanks for the pointer (never knew that)!  I'll update.

Reply:
Done